docroot/modules/custom/yukon_migrate/src/Plugin/migrate/process/Links.php

```diff
@@ -42,6 +42,7 @@ public function __construct(array $configuration, $plugin_id, $plugin_definition
       'yukon_migrate_basic_page_translations',
       'yukon_migrate_multi_step_page',
       'yukon_migrate_multi_step_page_translations',
+      'yukon_migrate_sub_heading',
```

Comment:
This array contains the list of node migrations where the plugin will extract the node id from the link string and then will search for the new node id and replace the string with the new id. 
You are adding the migration of paragraphs to this process plugin and this looks wrong.

Can you please explain why did you add this migration to this list?

Reply:
I don't think this update belongs to this pull request anyway as it should focus on configuration only, not migration.

Reply:
Removed all other files. This is now config only.